transports/notifications/index.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * index.js
+ * Module for transferring messages through desktop notifications
+ */
+'use strict';
+
+/**
+ * Importing modules
+ */
+const io = require('../../includes/io.js'),
+	util = require('../../includes/util.js'),
+	Transport = require('../transport.js'),
+	notifier = require('node-notifier'),
+	opn = require('opn');
+let    config,
```

Comment:
I'd rather not have variables outside of the class if they aren't constants. `config` is already `this.config` inside the Notifications class, and `diff` could be converted to `this.diff`